openshift-install-powervs

```diff
@@ -1305,12 +1298,24 @@ function main {
       IBM_CLI_OS="osx"
       OC_CLI_OS="mac"
       ARCHIVE_TYPE="tar.gz"
+      OS_VERSION=$(sw_vers -productVersion 2>/dev/null || echo "")
```

Comment:
This check should be moved up. Should error out initially for Mac.

Reply:
Resolved.